wm/frame.go

```diff
@@ -397,7 +411,10 @@ func (f *frame) updateTitle() {
 }
 
 func (f *frame) borderWidth() uint16 {
-	return f.client.wm.scaleToPixels(wmTheme.BorderWidth)
+	if f.client.Decorated() {
```

Comment:
This logic seems the wrong way round. The main content of a function should be at the top level.
So if !decorated { return 0 } would read more logically